### ISSUE TITLE
[Feature]: Make font size configurable

### ISSUE THREAD

**Issue by MelleD** [neponn/ring-tile-card#34]
### Is your feature request related to a problem? Please describe.

Currently, the font size in the application is fixed. To improve usability and accessibility, users should be able to customize the font size according to their preference.

See line https://github.com/neponn/ring-tile-card/blob/main/src/rt-info.js#L59

### Describe the solution you'd like

A property for the font-size would be an option

**Comment by CH-1967** [comment]
I would also appreciate it if this could be customized.
Especially bottom_element
